lldb/include/lldb/Target/Language.h

```diff
@@ -363,6 +363,14 @@ class Language : public PluginInterface {
     return false;
   }
 
+  /// Returns a boolean indicating whether two symbol contexts correspond to
+  /// the same function. If the plugin has no opinion, it should return nullopt.
+  virtual std::optional<bool>
+  AreEquivalentFunctions(const SymbolContext &sc1,
```

Comment:
equivalent functions sounds quite vague to my mind. Given that this function operates on symbol contexts, and is called from `IsEquivalentContext`, I think this would be better off named `AreEquivalentContexts`.

Reply:
Yup, I agree. I'll just wait to see if others chime in and then update the patch

Reply:
It would be nice to somehow indicate the comparison w.r.t. which these functions/contexts are equivalent.  "Equivalent" makes it sound like maybe I could use one in place of the other generally (like maybe I can call either one of them???)  `AreEqualForFrameComparison` is a little verbose, but tells you what statement we are actually making.

Reply:
Yup, that name looks a lot more expressive.

Reply:
Renamed it